web/server/entrypoint_test.go

```diff
@@ -405,6 +418,16 @@ func TestTunnelE2E(t *testing.T) {
 		test.That(t, destListener.Close(), test.ShouldBeNil)
 	}()
 
+	// Start mock "destination" listener, even if we don't intend on actually accepting any messages.
+	// This is because windows doesn't seem to allow for dialing to ports there aren't listeners on.
+	timeoutDestPort := 65534
+	timeoutDestListenerAddr := net.JoinHostPort("127.0.0.1", strconv.Itoa(timeoutDestPort))
+	timeoutDestListener, err := net.Listen("tcp", timeoutDestListenerAddr)
```

Comment:
kinda dumb but windows sometimes stop you from connecting to a port that doesn't have listeners?